modules/reference/appvms/comms.nix

```diff
@@ -14,15 +14,34 @@
     then config.ghaf.reference.services.dendrite
     else false;
 in {
-  name = "element";
+  name = "comms";
 
-  packages =
+  packages = let
+    slackWebviewScript = pkgs.writeShellScriptBin "slack-webview" ''
+      URL="$1"
+      echo "Opening $URL" | systemd-cat -p info
+      exec ${pkgs.eolie}/bin/eolie "$URL"
+    '';
+
+    xdgSlackWebviewItem = pkgs.makeDesktopItem {
+      name = "slack-handler";
+      desktopName = "Slack Handler";
+      exec = "${slackWebviewScript}/bin/slack-webview %u";
+      mimeTypes = ["x-scheme-handler/https"];
+    };
+  in
     [
       pkgs.element-desktop
       pkgs.element-gps
+      pkgs.slack
       pkgs.gpsd
       pkgs.tcpdump
       pkgs.pulseaudio
+      pkgs.chromium
```

Comment:
why chromium?

Reply:
i was testing the chromium first, i'll remove it